lib/ruby_lsp/ruby_lsp_rails/support/rails_document_client.rb

```diff
@@ -0,0 +1,122 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "net/http"
+
+module RubyLsp
+  module Rails
+    module Support
+      class RailsDocumentClient
+        RAILS_DOC_HOST = "https://api.rubyonrails.org"
+        SUPPORTED_RAILS_DOC_NAMESPACES = T.let(
```

Comment:
Adding a space to be consistent with the next constant:

```suggestion

        SUPPORTED_RAILS_DOC_NAMESPACES = T.let(
```